src/plugins/plan/PlanViewProvider.js

```diff
@@ -25,7 +25,9 @@ import Vue from 'vue';
 
 export default function PlanViewProvider(openmct) {
     function isCompactView(objectPath) {
-        return objectPath.find(object => object.type === 'time-strip') !== undefined;
+        let isChildOfTimeStrip = objectPath.find(object => object.type === 'time-strip');
```

Comment:
Looks good but wondering  about following change to avoid extra checks,
```
if (openmct.router.isNavigatedObject(objectPath)) return false;
....
```

Reply:
No we need both checks so I'd rather keep it as is.